Dockerfile

```diff
@@ -1,34 +1,20 @@
-## Dependency build environment
-FROM golang:1.11.2-stretch as build
-
-WORKDIR /go/src/
-
-RUN apt-get update && \
-    apt-get install -y --no-install-recommends libltdl-dev && \
-    apt-get clean
-
-RUN git clone https://github.com/square/ghostunnel.git $GOPATH/src/github.com/square/ghostunnel && \
-    cd  $GOPATH/src/github.com/square/ghostunnel && \
-    git checkout -b v1.3.0 && \
-    make
-
-## Runtime container
-FROM python:3.7-slim-stretch
+# docker run --net=host --cap-add=NET_ADMIN --cap-add=SYS_ADMIN  -v /sys/fs/cgroup:/sys/fs/cgroup:ro --security-opt seccomp:unconfined -v ~/Documents/GreatFruit/WoTT:/usr/src/app -Pit mydebian
```

Comment:
Let's move this to a bash script instead.

Reply:
I'll remove this, this command is exactly what docker-compose does.

Reply:
Ah right.